### PR TITLE
Refactor private methods to src package

### PR DESCRIPTION
**Pull request by st3v3nhunt**
https://eaflood.atlassian.net/browse/PSD-661

To do:
- [x] Remove all `//public` and `//private` comments
- [x] Ensure all variables are scoped to the function (where possible)
- [x] Update `README` or the individual documentation for each of the global vars
- [x] Should all of the methods within each global vars script just wrap a call to the src class?

### CODE REVIEW

File: src/uk/gov/defra/ffc/Helm.groovy [hunk 1]
```diff
@@ -0,0 +1,79 @@
+package uk.gov.defra.ffc
+
+class Helm implements Serializable {
+  static def writeUrlIfIngress(ctx, deploymentName) {
+    ctx.sh("kubectl get ingress -n $deploymentName -o json --ignore-not-found | jq '.items[0].spec.rules[0].host // empty' | xargs --no-run-if-empty printf 'Build available for review at https://%s\n'")
+  }
+
+  static def addHelmRepo(ctx, repoName, url) {
+    ctx.sh("helm repo add $repoName $url")
+    ctx.sh("helm repo update")
```
Comment (markysoft) [inline src/uk/gov/defra/ffc/Helm.groovy:10]: We generally prefer single quotes for non-interpolated strings

File: src/uk/gov/defra/ffc/Release.groovy [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+   * - the version tag in SemVer format to check for on GitHub e.g 1.0.0
+   * - the repository name to check
+   * - the GitHub connection token secret text
+   */
+  private static def exists(ctx, versionTag, repoName, token){
+    try {
+      return ctx.sh(returnStdout: true, script: "curl -s -H 'Authorization: token $token' https://api.github.com/repos/DEFRA/$repoName/releases/tags/$versionTag | jq '.tag_name'").trim().replaceAll (/"/, '') == versionTag ? true : false
+    }
+    catch(Exception ex) {
+      ctx.echo("Failed to check release status on github")
```
Comment (markysoft) [inline src/uk/gov/defra/ffc/Release.groovy:20]: We generally prefer single quotes for non-interpolated strings

File: src/uk/gov/defra/ffc/Terraform.groovy [hunk 3]
```diff
@@ -0,0 +1,112 @@
+package uk.gov.defra.ffc
+
+class Terraform implements Serializable {
+  static def String generateTerraformInputVariables(serviceCode, serviceName, serviceType, prCode, queuePurpose, repoName) {
+    def Map inputs = [service: [code: serviceCode, name: serviceName, type: serviceType], pr_code: prCode, queue_purpose: queuePurpose, repo_name: repoName];
+    return Utils.mapToString(inputs).join("\n")
```
Comment (markysoft) [inline src/uk/gov/defra/ffc/Terraform.groovy:6]: Could this be in single quotes?

File: src/uk/gov/defra/ffc/SecretScanner.groovy [hunk 4]
```diff
@@ -0,0 +1,4 @@
+package uk.gov.defra.ffc
+
+class SecretScanner implements Serializable {
+}
```
[the remark this answers is not in the document]
Comment (st3v3nhunt) [reply src/uk/gov/defra/ffc/SecretScanner.groovy]: File has been removed.

**Pull request merged**